### ISSUE TITLE
Bug: EzStatus use="success"  uses old ezCater green color

### ISSUE THREAD

**Issue by morgannegagne** [ezcater/recipe#717]
## Describe the bug

The success EzStatus uses the old ezCater green. The marketplace team is working on using the new ezCater green everywhere (we currently have a mix of old and new greens and it doesn't look great).  This isn't necessarily a bug, but something we might want to consider updating to have consistent branding across the board. 

This is the new WIP version of the menu page in store-next
![Screen Shot 2021-11-05 at 3 11 17 PM](https://user-images.githubusercontent.com/32595022/140566096-e3fbe088-b42c-4dfc-ad6d-8712927a7378.png)

Here is the current menu page with different versions of green
![Screen Shot 2021-11-05 at 3 15 32 PM](https://user-images.githubusercontent.com/32595022/140566208-5d1364a8-f57d-4248-b65f-8bf23f87ea1b.png)

Also the success EzStatus doesn't meet the WCAG accessibility contrast recommendation - https://coolors.co/contrast-checker/609b3b-f3f8eb. Here is the option that we are going with for the current menu page in store-react
![Screen Shot 2021-11-05 at 3 11 58 PM](https://user-images.githubusercontent.com/32595022/140566139-5b0991a3-8f89-4c2d-abca-0e6873fc9121.png)

## Steps To Reproduce

1.
2.

Link to code example:

**Comment by danidewitt** [comment]
@morgannegagne absolutely. now that we have full-time designers for store and ezmanage, we should be able to sync and prioritize these kind of branding changes more easily
